src/app/root/root.component.html

```diff
@@ -10,14 +10,12 @@
       [options]="notificationOptions">
     </ds-notifications-board>
     <main class="main-content">
-      <div class="container">
-        <ds-themed-breadcrumbs></ds-themed-breadcrumbs>
-      </div>
+      <ds-themed-breadcrumbs></ds-themed-breadcrumbs>
 
-      <div class="container" *ngIf="isLoading">
-        <ds-loading message="{{'loading.default' | translate}}"></ds-loading>
+      <div class="container d-flex justify-content-center align-items-center h-100" *ngIf="isLoading">
+        <ds-loading [showMessage]="false"></ds-loading>
       </div>
-      <router-outlet></router-outlet>
+      <router-outlet *ngIf="!isLoading"></router-outlet>
```

Comment:
@atarix83 While I think this is a good change, I think it's a better idea to _hide_ the router-outlet while loading. `*ngIf` doesn't render it at all, which means that only after isLoading turns false all the components that depend on the route can run _their_ onInit methods etc.

If we hide it instead, that can happen in the background. For the user it looks the same.
```suggestion
      <router-outlet [class.d-none]="isLoading"></router-outlet>
```